patches/libtock-rs/08-buffered-console.patch

```diff
@@ -0,0 +1,95 @@
+diff --git a/src/console.rs b/src/console.rs
+index ecd7ad1..785de01 100644
+--- a/src/console.rs
++++ b/src/console.rs
+@@ -16,33 +16,58 @@ mod allow_nr {
+     pub const SHARE_BUFFER: usize = 1;
+ }
+ 
++const BUFFER_SIZE: usize = 64;
```

Comment:
**Note:** this buffer size seems to work fine.

If I use a larger buffer such as 1024 bytes on an nRF52840-DK with Segger RTT debugging, the output gets somehow scrambled.

Reply:
I could reproduce this bug more generally as https://github.com/tock/tock/issues/1697.